hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_exchange.py

```diff
@@ -99,9 +100,25 @@ def __init__(
         self._order_book_manager_connected = False
         asyncio.create_task(self._initialize_order_manager())
 
+        self._token_accounts: Dict[str, str] = {}
+        asyncio.create_task(self._initialize_token_accounts())
+
         super().__init__(client_config_map)
         self.real_time_balance_update = False
 
+    async def _initialize_token_accounts(self):
+        await self._provider_1.connect()
```

Comment:
think we have some other discussion about this elsewhere. fine for now, let's address that all together elsewhere